templates/managed/kafka.html

```diff
@@ -141,6 +141,20 @@ <h3>
 </section>
 
 <section class="p-strip">
+  <div class="row" id="webinar">
+    <div class="col-5">
+      <div>
+        <h3>Kafka can be challenging. Learn&nbsp;how we simplify it.</h3>
+        <p><a href="https://www.brighttalk.com/webcast/6793/410402" class="js-invoke-modal p-link--external">Watch the webinar on BrightTALK</a></p>
```

Comment:
```suggestion
        <p><a href="https://www.brighttalk.com/webcast/6793/410402" class="p-link--external">Watch the webinar on BrightTALK</a></p>
```
it shouldn't open the contact us modal